src/components/input/inputField.tsx

```diff
@@ -61,8 +59,15 @@ interface Props extends InputHTMLAttributes<HTMLInputElement> {
   hasError?: string
   disabled?: boolean
   required?: boolean
-  autoComplete?: string
-  onChange?: (e: ChangeEvent<HTMLInputElement>) => void
+  onChange?: (
+    e:
+      | ChangeEvent<HTMLInputElement>
+      | {
```

Comment:
This looks a bit like a workaround to me, what's the reason why we need to handle a custom interface and cant we update the source so where we call change with a different event instead to avoid having to support a custom interface?